README.md

```diff
@@ -195,3 +195,7 @@ overwriting default console styles, breaking the layout of existing pages. The
 best practice is to prefix your CSS classnames with your plugin name to avoid
 conflicts. Please don't disable these rules without understanding how they can
 break console styles!
+
+### Local Development Troubleshooting
+1. Disable cache. Select 'disable cache' in your browser's DevTools > Network > 'disable cache'. Or use private/incognito mode in your browser.
+2. Enable higher log verbosity by setting `-log-verbosity=1` when starting the plugin backend
```

Comment:
```suggestion
2. Enable higher log verbosity by setting `-log-level=1` when starting the plugin backend
```

Reply:
Also this is now based on the log level, not a number. This should be `-log-level=trace`

Reply:
Updated to `-log-level=trace` and linked logrus docs on the different log levels.